onnx/defs/nn/defs.cc

```diff
@@ -17,9 +17,11 @@ const char* auto_pad_doc =
     "auto_pad must be either SAME_UPPER, SAME_LOWER or VALID. Where "
```

Comment:
Shouldn't 'NOTSET' be listed among the valid values? It sounds like it is one of the allowed values.